sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -358,52 +358,35 @@ def list_available_area_codes(
         locality=None, # type: str
         **kwargs # type: Any
     ):
+        # type: (...) -> ItemPaged[PhoneNumberAreaCode]
+        """Gets the list of available toll free area codes.
 
+        Gets the list of available area codes.
+        If receives a locality it lists the geographic area codes, otherwise it gets the toll free ones.
+
+        :param country_code: The ISO 3166-2 country/region two letter code, e.g. US. Required.
+        :type country_code: str
+        :keyword skip: An optional parameter for how many entries to skip, for pagination purposes. The
+        default value is 0. Default value is 0.
+        :paramtype skip: int
+        :param locality: The name of locality in which to search for the area code. e.g. Seattle.
+        This is required if the phone number type is Geographic. Default value is None.
+        :type locality: str
+        :param administrative_division: The name of the state or province in which to search for the
```

Comment:
This should be `:keyword` instead of `:param`. 
```suggestion
        :keyword administrative_division: The name of the state or province in which to search for the
```